test/unit/api/core.js

```diff
@@ -157,6 +157,33 @@ describe('Core API', function () {
         })
     })
 
+    it('sign for contract assets', () => {

```

Comment:
do use SC or smart contract instead of just contract. An normal address is a normal contract.